package.json

```diff
@@ -11,6 +11,7 @@
     "build:watch": "concurrently \"yarn build --watch\" \"node --inspect=0.0.0.0:9229 --watch dist/app.js\"",
     "predev": "docker compose build migrate fake-gcs",
     "dev": "docker compose up activitypub nginx -d --build",
+    "dev:ghost": "docker compose up activitypub -d --build",
```

Comment:
@sagzy I was thinking of changing this to:

- `dev` - `docker compose up activitypub -d --build`
- `dev:standalone` - `docker compose up activitypub nginx -d --build`

but didn't want to break muscle memory 😅